mozilla-release/browser/components/preferences/in-content/privacy.xul

```diff
@@ -735,7 +745,7 @@
   <html:h1 data-l10n-id="collection-header"/>
 </hbox>
 
-<groupbox id="dataCollectionGroup" data-category="panePrivacy" hidden="true">
+<groupbox id="dataCollectionGroup" data-category="panePrivacy" hidden="true" class="formatBoxSize">
```

Comment:
@tsl143 I am not sure that we need this class here. Doesn't it work without it?

Reply:
This is the base of whole PR hahaha 
Refer https://bugzilla.mozilla.org/show_bug.cgi?id=1202492